gmso/core/topology.py

```diff
@@ -546,6 +578,22 @@ def update_atom_types(self):
                 site.atom_type = self._atom_types[site.atom_type]
         self.is_typed(updated=True)
 
+    def add_pairpotential(self, pair_potential_type):
+        """Add a new pair potential into the topology that overrides the combin        ation rule.
+
+        See Also:
+        ---------
+        gmso.PairPotentialType: A description of nonbonded interaction between 2        atomtypes
+        """
+        if not isinstance(pair_potential_type, PairPotentialType):
+            raise GMSOError(
+                'Non pairpotential type {}'.format(pair_potential_type))
+
+        elif pair_potential_type not in self._pairpotential_types:
+            pair_potential_type.topology = self
```

Comment:
Since it looks like a `PairPotentialType` is a descripton of the nonbondedinteraction between 2 atomtypes, it might help to check if the `AtomTypes` exist in the topology before adding a `PairPotentialType` between the.